OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDatesPageFragment.kt

```diff
@@ -254,21 +254,25 @@ class CourseDatesPageFragment : OfflineSupportBaseFragment(), BaseFragment.Permi
     }
 
     private fun askForCalendarSync() {
-        val title: String = ResourceUtil.getFormattedString(resources, R.string.title_add_course_calendar, AppConstants.COURSE_NAME, calendarTitle).toString()
-        val message: String = ResourceUtil.getFormattedString(resources, R.string.message_add_course_calendar, AppConstants.COURSE_NAME, calendarTitle).toString()
-
-        val alertDialog = AlertDialogFragment.newInstance(title, message, getString(R.string.label_ok),
-                { _: DialogInterface, _: Int ->
-                    trackCalendarEvent(Analytics.Events.CALENDAR_ADD_OK, Analytics.Values.CALENDAR_ADD_OK)
-                    insertCalendarEvent()
-                },
-                getString(R.string.label_cancel),
-                { _: DialogInterface?, _: Int ->
-                    trackCalendarEvent(Analytics.Events.CALENDAR_ADD_CANCEL, Analytics.Values.CALENDAR_ADD_CANCEL)
-                    binding.switchSync.isChecked = false
-                })
-        alertDialog.isCancelable = false
-        alertDialog.show(childFragmentManager, null)
+        if (environment.calendarPrefs.isSyncAlertPopupDisabled(calendarTitle.replace(" ", "_"))) {
```

Comment:
IMO, it better to directly use `courseData.course.name` instated of `calendarTitle.replace(" ", "_")`.

same with the other location too.

Reply:
as we are using `courseTitle` as the main key everywhere that's why I prefer to use this here as well